a3p-integration/proposals/b:enable-orchestration/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "agoricProposal": {
+    "type": "/agoric.swingset.CoreEvalProposal",
+    "source": "subdir",
+    "sdk-generate": [
+      "vats/init-orchestration.js"
+    ]
+  },
+  "type": "module",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@agoric/synthetic-chain": "^0.1.0",
+    "ava": "^5.3.1"
+  },
+  "ava": {
+    "concurrency": 1,
+    "serial": true,
```

Comment:
@0xpatrickdev you mentioned using `serial` in the starship work; apparently that's pretty normal for this sort of thing.